.github/workflows/go-test.yaml

```
@@ -225,6 +225,23 @@ jobs:
           kubectl -n test-cluster scale deployment rook-ceph-osd-0 --replicas 0
           kubectl rook-ceph --operator-namespace test-operator -n test-cluster rook purge-osd 0 --force
 
+      - name: List CRDS
```

nit: Convert this in a few places: CRDS --> CRDs
```suggestion
      - name: List CRDs
```